libs/template/cli/bin/cli.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { vi } from 'vitest';
```

Comment:
We don't need this import

Reply:
`vi` is not global, if I remove it I get `Cannot find name 'vi'.` so it should be imported.
Also global typings are from jest which are not quite correct so I also needed to import `expect` to make sure it's properly typed.